.github/workflows/build.yml

```diff
@@ -79,6 +80,50 @@ jobs:
           asset_name: exograph-${{matrix.target}}.zip
           tag: ${{ github.ref }}
 
+  build-aws-lambda:
```

Comment:
could this be folded into the above matrix by having a matrix field of the list of packages to be built and have the arch specified for each one too?

Reply:
I thought about it, but it felt that it doesn't fit nicely. In normal case, we have to copy `exo`+`exo_server`, while in case of aws, we have to do just `bootstrap`.

I am also unsure so far if `ubuntu-20.04` built version will work with aws-lambda (trying it out separately). If it does, I will give the matrix idea another thought (with a few matrix variables).